test-asset-generator/transformers/types/marketing-authorisation.js

```diff
@@ -1,13 +1,14 @@
 export default class MarketingAuthorisation {
```

Comment:
It would be better to have a factory function here that consumed the correct class and spat out the data rather than having a class consume a class which consumes a class.

Reply:
Yep can sort this as tech debt.